### DISCUSSION TITLE
useSession returns only user.name and user.email

### DISCUSSION THREAD

**Discussion by nakamigo** [nextauthjs/next-auth#3883] [Help]
I'm using GitHub provider with jwt strategy:
```js
export default NextAuth({
  adapter: PrismaAdapter(prisma),
  secret: process.env.NEXTAUTH_SECRET,
  providers: [
    GithubProvider({
      clientId: process.env.GITHUB_ID,
      clientSecret: process.env.GITHUB_SECRET,
    }),
  ],
  session: {
    strategy: "jwt",
  },
});
```
I'm expecting to get id, name, email and image, according to [this snippet](https://github.com/nextauthjs/next-auth/blob/main/packages/next-auth/src/providers/github.js#L34-L41)
Instead useSession returns this:
```js
{
  "user": {
    "name": "User",
    "email": "user@gmail.com"
  },
  "expires": "2022-03-10T14:15:23.739Z",
}
```
When I login for the first time I can see all the data returned from GitHub.
Also, JWT callback returns more full information about the token but no image.

How can I get information about the image or it's available only with database sessions?

**Comment by malikk908** [comment]
facing same issue, did you solve it?

**Comment by jpcmf** [comment]
I'm facing the same issue. But in my case it's only returns user email.
